src/snowflake/snowpark/modin/plugin/_internal/telemetry.py

```diff
@@ -513,14 +535,17 @@ def __new__(
                 snowflake.snowpark.modin.pandas.window.Rolling]:
                 The modified class with decorated methods.
         """
+        wrapped_attrs: dict[Any, Any] = {}
         for attr_name, attr_value in attrs.items():
             if callable(attr_value) and (
                 not attr_name.startswith("_")
                 or (attr_name in TELEMETRY_PRIVATE_METHODS)
             ):
-                attrs[attr_name] = snowpark_pandas_telemetry_method_decorator(
-                    attr_value
+                attrs[attr_name] = wrapped_attrs.get(
```

Comment:
So this ensures that the wrapped method is not overwritten with a second instance of the dunder method? Were we seeing issues where _add_ was overwritten? I'm not sure I totally understand the problem.

Reply:
We weren't seeing this issue, but I added this when I thought that that was the original issue with `__add__` - I'll go ahead and remove!